README.md

```diff
@@ -7,7 +7,7 @@ Scripts used to generate metrics for the Server and Foundations Teams.
 ## How to Run
 
 ### Dependencies
-```sudo apt-get install python3-distro-info```
+```sudo apt-get install python3-distro-info python3-influxdb python3-prometheus-client```
```

Comment:
Thinking about this more strictly speaking you don't need these additional packages to test the scripts, its only if you want to write to the local testing environment so maybe this should move to the local testing section.